### DISCUSSION TITLE
Help on customlinks/nbshell that worked with older netbox

### DISCUSSION THREAD

**Discussion by phurrelmann** [netbox-community/netbox#10749] [General]
Hi all,

we've updated an quite old netbox instance from 2.7.12 to 3.3.5 and we have failing custom-links that worked in 2.7.12.
I tried to debug and adjust the link (most probably there were changes in netbox), but I fail to resolve the issue.

The custom-link in question filters the ip-addresses assigned to an vc_interface:

`https://{{ obj.vc_interfaces.get(mgmt_only=True).ip_addresses.get(family=4).address.ip }}/`

If I try to debug that via nbshell I'm stuck. The get() and filter() functions work on the vc_interfaces object, but I can't get any details beyond ip_addresses().

```
>>> d = Device.objects.get(name='example_device')
>>> mgmtif = d.vc_interfaces().get(mgmt_only=True)      
>>> mgmtif.ip_addresses().get()
Traceback (most recent call last):
  File "<console>", line 1, in <module>
TypeError: create_generic_related_manager.<locals>.GenericRelatedObjectManager.__call__() missing 1 required keyword-only argument: 'manager'

```

What am I doing wrong?

**Comment by kkthxbye-code** [comment]
```
>>> i = Interface.objects.first()
>>> i.ip_addresses.first()
<IPAddress: 10.10.10.10/32>
>>> i.ip_addresses().first()
Traceback (most recent call last):
  File "<console>", line 1, in <module>
TypeError: create_generic_related_manager.<locals>.GenericRelatedObjectManager.__call__() missing 1 required keyword-only argument: 'manager'
```

Call it without parenthesis.
